src/map-marker/__tests__/map-style.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+type GeoJSONPointFeature = {
+  type: 'Feature',
+  geometry: {
+    type: 'Point',
+    coordinates: [number, number],
+  },
+};
+
+export function getMapStyle(
```

Comment:
Moved the source and the layer configurations for debug points into this separate file